src/server/game/Maps/Map.cpp

```diff
@@ -2596,6 +2600,104 @@ ZLiquidStatus Map::getLiquidStatus(float x, float y, float z, uint8 ReqLiquidTyp
     return result;
 }
 
+void Map::getFullTerrainStatusForPosition(float x, float y, float z, PositionFullTerrainStatus& data, uint8 reqLiquidType) const
```

Comment:
method name? make it start with uppercase (you can leave the one in vmapmanager as-is)